diff --git a/autoload/tsuquyomi/config.vim b/autoload/tsuquyomi/config.vim
index 48c075e..9a500ca 100644
--- a/autoload/tsuquyomi/config.vim
+++ b/autoload/tsuquyomi/config.vim
@@ -110,6 +110,7 @@ function! tsuquyomi#config#getVersion()
     return s:tss_version
   endif
   let l:cmd = substitute(tsuquyomi#config#tsscmd(), 'tsserver', 'tsc', '')
+  let l:cmd = substitute(l:cmd, "\\", "/", "g")
   let out = s:Process.system(l:cmd.' --version')
   let pattern = '\vVersion\s+(\d+)\.(\d+)\.(\d+)-?([^\.\n\s]*)'
   let matched = matchlist(out, pattern)
