diff --git a/build.py b/build.py
index 0d7d5db2..37e90b27 100644
--- a/build.py
+++ b/build.py
@@ -222,7 +222,9 @@ def CustomPythonCmakeArgs():
 
 def GetGenerator( args ):
   if OnWindows():
-    if args.msvc == 14:
+    if args.msvc == 15:
+      generator = 'Visual Studio 15 2017'
+    elif args.msvc == 14:
       generator = 'Visual Studio 14'
     elif args.msvc == 12:
       generator = 'Visual Studio 12'
@@ -254,7 +256,7 @@ def ParseArguments():
   parser.add_argument( '--system-boost', action = 'store_true',
                        help = 'Use the system boost instead of bundled one. '
                        'NOT RECOMMENDED OR SUPPORTED!')
-  parser.add_argument( '--msvc', type = int, choices = [ 11, 12, 14 ],
+  parser.add_argument( '--msvc', type = int, choices = [ 11, 12, 14, 15 ],
                        default = 14, help = 'Choose the Microsoft Visual '
                        'Studio version (default: %(default)s).' )
   parser.add_argument( '--tern-completer',
